geode-assembly/src/acceptanceTest/resources/org/apache/geode/client/sni/docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
     container_name: 'haproxy'
     image: 'haproxy:2.1'
     ports:
-      - "15443"
+      - "15443:15443"
```

Comment:
binding to a well known port, intentionally

Reply:
This will cause the test to be flaky if something else has this port. AvailablePortHelper would help here.

Reply:
this has been addressed now @upthewaterspout